ynr/apps/moderation_queue/review_required_helper.py

```diff
@@ -216,6 +216,25 @@ def needs_review(self):
             return self.Status.UNDECIDED
 
 
+class RevertedEdits(BaseReviewRequiredDecider):
+    def review_description_text(self):
+        return "Too many reverted edits in 24 hours"
+
+    def needs_review(self):
+        from candidates.models import LoggedAction
+
+        if self.logged_action.action_type.PERSON_REVERT:
```

Comment:
I think this is always `True`? You want `self.logged_action.action_type = PERSON_REVERT` (or whatever the path for `PERSON_REVERT` is, Maybe `self.logged_action.action_type = self.logged_action.action_type.PERSON_REVERT`?)